common/lib/xmodule/xmodule/js/src/sequence/display.coffee

```diff
@@ -153,7 +165,17 @@ class @Sequence
       new_position = $(event.currentTarget).data('element')
 
     if (1 <= new_position) and (new_position <= @num_contents)
-      Logger.log "seq_goto", old: @position, new: new_position, id: @id
+      is_bottom_nav = $(event.target).closest('nav[class="sequence-bottom"]').length > 0
+      if is_bottom_nav
+        widget_placement = 'bottom'
+      else
+        widget_placement = 'top'
+      Logger.log "edx.ui.lms.sequence.tab_selected",
```

Comment:
Suggestion: we should put a comment here (and at other locations where events are now being emitted with a different name) about this being the location that we used to emit "seq_goto" or something similar.

A consumer of the event log may want to find the exact location in the code where the event originated from to better understand the values in it etc. Having that string appear here would help a lot with that search process.